### PR TITLE
Checkbox grouping label Issue 604

### PR DESCRIPTION
**Pull request by jongund**
We have added the role=group and aria-labelledby to reference the grouping label.
We have also updated the example documentation to include the role=group and aria-labelledby additions.

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[branch]
```
Comment (mcking65) [inline .gitconfig]: Please remove this file from the PR. If you want to set this option, best to do it in .git/config (your local repo config file)

**Comment by jongund** [comment]
Matt,

Git config should be removed now.

Jon

File: examples/checkbox/checkbox-1/checkbox-1.html [hunk 2]
```diff
@@ -37,8 +37,8 @@ <h2 id="ex_label">Example</h2>
       <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
 
       <div  id="ex1">
-        <h3>Sandwich Condiments</h3>
-        <div>
+        <h3 id="id-group-label">Sandwich Condiments</h3>
+        <div role="group" aria-labelledby="id-group-label">
```
Comment (mcking65) [inline examples/checkbox/checkbox-1/checkbox-1.html]: Sorry I missed this before ... the states and properties table says that the group role is on the UL element. It is actually on a div here. I'm not sure which is better. The advantage of the div is the list semantics that says there are 4 checkboxes is preserved. The disadvantage is that is just more verbage on the page that may not be particularly valuable. I could go either way. We just need the documentation and the code to be consistent.

File: examples/checkbox/checkbox-1/checkbox-1.html [hunk 3]
```diff
@@ -101,6 +101,35 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
         </thead>
         <tbody>
             <tr>
+              <th scope="row"></th>
+              <td></td>
+              <td><code>h3</code></td>
+              <td>
+                <ul>
+                  <li>Provides a grouping label for the group of checkboxes.</li>
+                </ul>
```
Comment (mcking65) [inline examples/checkbox/checkbox-1/checkbox-1.html]: This is the entry that is inconsistent with the code.

**Comment by jongund** [comment]
Matt,

I updated the documentation to reference DIV element.

I think we should document in the example why we want the list semantics are not being overridden.

Jon

**Pull request merged**